From 0f88648a5be92331b1de7055302bd83d6e3d30ca Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Tue, 1 Aug 2017 11:41:58 +0100
Subject: [PATCH] compiler-rt: Determine COMPILER_RT_SUPPORTED_ARCH even if
 MACOSX_VERSION_MIN_FLAG passed

For some reason, the block of code that determines which arches to build compiler-rt
for was getting elided if MACOSX_VERSION_MIN_FLAG was set (i.e. if CXXFLAGS contains
-mmacosx-version-min=${MACOSX_DEPLOYMENT_TARGET})

This means that compiler-rt does not get built at all.
---
 cmake/config-ix.cmake | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/cmake/config-ix.cmake b/cmake/config-ix.cmake
index 192c9b0..8682289 100644
--- a/cmake/config-ix.cmake
+++ b/cmake/config-ix.cmake
@@ -268,7 +268,7 @@ if(APPLE)
   darwin_get_toolchain_supported_archs(toolchain_arches)
   message(STATUS "Toolchain supported arches: ${toolchain_arches}")
   
-  if(NOT MACOSX_VERSION_MIN_FLAG)
+  # if(NOT MACOSX_VERSION_MIN_FLAG)
     darwin_test_archs(osx
       DARWIN_osx_ARCHS
       ${toolchain_arches})
@@ -331,7 +331,7 @@ if(APPLE)
         endforeach()
       endif()
     endforeach()
-  endif()
+  # endif()
 
   # for list_intersect
   include(CompilerRTUtils)
-- 
2.8.4

